README.md

```diff
@@ -171,22 +173,23 @@ The same works for version 5.
 
 #### Upgrade from Agent v6 to Agent v7
 
-To upgrade from an already installed Agent v6 to Agent v7, you'll have to set the `agent_package_action` to `install` and we recommend to pin to a specific version:
+To upgrade from an already installed Agent v6 to Agent v7, you'll have to either:
+
+* Set `agent_major_version` to `7`, `agent_package_action` to `upgrade` and pin a specific v7 version as `agent_version` (recommended).
```

Comment:
note: as long as `agent_version` is set to a version, `agent_package_action` does not have to be set to `upgrade` (`install` is fine)

Reply:
Copy-paste error!